### PR TITLE
Styling the project and solving the "N + 1" issues

### PR DESCRIPTION
**Pull request by Haywayaheadshot**
# In this milestone and partners and I ensured the following:
- Styled the whole project to look similar to the wireframe provided in the requirements.
- Solved the "N + 1" issues to enhance our code performance.

### CODE REVIEW

File: app/controllers/recipes_food_controller.rb [hunk 1]
```diff
@@ -13,13 +13,13 @@ def show
...
 
-  def recipe_foods_params
-    params.require(:recipe_foods).permit(:food_id, :quantity)
-  end
-
   def destroy
     @recipe_food = RecipeFood.find(params[:recipe_food_id])
     @recipe_food.destroy
     redirect_back(fallback_location: root_path)
   end
```
Comment (Goruchie) [inline app/controllers/recipes_food_controller.rb:20]: Kindly make sure you use `.includes` to solve N + 1 problem, i.e.:
```ruby
@recipe_food = RecipeFood.includes(:foods, :recipes).find(params[:recipe_food_id])

**Review by Goruchie — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Haywayaheadshot 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@Goruchie ](https://github.com/Goruchie) when you comment so I can receive the notification or use slack with the same name._

File: app/controllers/recipes_food_controller.rb (hunk 1 above)
Comment (Goruchie) [inline app/controllers/recipes_food_controller.rb:20]: - [ ] Kindly make sure to use `.includes` to solve N + 1 error, i.e.:
```ruby
@recipe_food = RecipeFood.includes(:foods, :recipes).find(params[:recipe_food_id])

**Review by Goruchie — CHANGES_REQUESTED** [review]

File: app/controllers/recipes_food_controller.rb (hunk 1 above)
Comment (Goruchie) [inline app/controllers/recipes_food_controller.rb:20]: - [ ] Kindly make sure to use `.includes` to solve N + 1 error, i.e.:
```ruby
@recipe_food = RecipeFood.includes(:foods, :recipes).find(params[:recipe_food_id])

**Comment by Haywayaheadshot** [comment]
@Goruchie Thanks for reviewing the project. I have fixed the errors

**Review by Goruchie — APPROVED** [review]
Hi @Haywayaheadshot 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 
* Great following of the changes required by the last review 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**